SingularityUI/app/models/Request.coffee

```diff
@@ -146,8 +149,9 @@ class Request extends Model
                     @run( @data.commandLineInput ).done callback( @data )
                     return true
 
-            afterOpen: -> 
+            afterOpen: => 
                 $('#filename').val localStorage.getItem('taskRunRedirectFilename')
+                $('#commandLineInput').val localStorage.getItem(@localStorageCommandLineInputKeyPrefix + @id)
```

Comment:
`@id` comes from the request model (in addition to the attribute id - `@get("id")`). I'm having no trouble in my tests.